### PR TITLE
Add missing iterable type hints

### PR DESCRIPTION
**Pull request by Slamdunk**

### CODE REVIEW

File: .github/workflows/integrate.yaml [hunk 1]
```diff
@@ -275,7 +275,9 @@ jobs:
         run: "vendor/bin/phpstan analyse --no-progress --error-format=checkstyle | cs2pr"
 
       - name: "Run PSalm for type-coverage"
-        run: "vendor/bin/psalm --no-progress --shepherd --show-info=false --stats"
+        run: |
+          vendor/bin/psalm --no-progress --show-info=false > /dev/null || true
+          vendor/bin/psalm --no-progress --shepherd --show-info=false --stats
```
Comment (Slamdunk) [inline .github/workflows/integrate.yaml:280]: https://github.com/vimeo/psalm/issues/3901

File: bin/phpunit-wrapper.php [hunk 2]
```diff
@@ -16,18 +19,19 @@
...
-$lastExitCode = 0;
-$rand = rand ( 0 , 999999 );
-$uniqueTestToken = getenv("UNIQUE_TEST_TOKEN") ?: "no_unique_test_token";
-$testToken = getenv("TEST_TOKEN") ?: "no_test_token";
-$filename = "paratest_t-{$testToken}_ut-{$uniqueTestToken}_r-{$rand}.log";
-$path = sys_get_temp_dir()."/".$filename;
-$loggingEnabled = getenv("PT_LOGGING_ENABLE");
-$logInfo = function(string $info) use ($path, $loggingEnabled){
-    if(!$loggingEnabled){
+$commands        = fopen('php://stdin', 'r');
```
Comment (andreasschroth) [inline bin/phpunit-wrapper.php:22]: I really don't like this alignment. It's not widespread and makes no sense, as it leads to unnecessary changes and diffs. E.g. when a longer variable is added here, the spaces need to be adjusted in all other lines, leading to unnecessary noise.

**Comment by andreasschroth** [comment]
Thanks a lot, good changes, I just added one comment.

File: bin/phpunit-wrapper.php (hunk 2 above)
Comment (andreasschroth) [reply bin/phpunit-wrapper.php:22]: Yeah, that would be great. I really think it's pointless, as I said, one change in another line might lead to changes in the surrounding lines, which imo is bad.
